codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/AuthGenerator.java

```diff
@@ -91,6 +115,13 @@ public void renderRbs() {
         LOGGER.fine("Wrote auth rbs module to " + rbsFile());
     }
 
+    private Object removeRbExtension(String s) {
```

Comment:
We do this elsewhere as well - I think we may want to create some sort of additional files utility

Reply:
I was thinking of adding a method to the writer, given a list of files, to write require relative.